app/helpers/select-metrics.js

```diff
@@ -0,0 +1,28 @@
+import { helper as buildHelper } from '@ember/component/helper';
+// import { htmlSafe } from '@ember/template';
+
+
+export function selectMetrics([ metrics, doi ]) {
```

Comment:
What is the function of this helper?

Reply:
I will get ride of this one, we don't need it after the last reviw